Sources/NIO/EventLoop.swift

```diff
@@ -725,6 +725,31 @@ public final class MultiThreadedEventLoopGroup: EventLoopGroup {
     private let shutdownLock: Lock = Lock()
     private var runState: RunState = .running
 
+    private static func runTheLoop(thread: NIOThread,
+                                   canEventLoopBeShutdownIndividually: Bool,
+                                   selectorFactory: @escaping () throws -> NIO.Selector<NIORegistration>,
+                                   initializer: @escaping ThreadInitializer,
+                                   _ callback: @escaping (SelectableEventLoop) -> Void) {
+        assert(NIOThread.current == thread)
+        initializer(thread)
+
+        do {
+            let l = SelectableEventLoop(thread: thread,
```

Comment:
Nit: `l` doesn't seem like a great name IMHO, could we at least go for `loop` or something?

Reply:
fixed.

Reply:
(was just moving code but should still use better variable names)